cdap-runtime-ext-dataproc/src/main/java/co/cask/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -296,12 +297,20 @@ public Capabilities getCapabilities() {
       }
     }
 
-    // If preferExternalIp has been set to true in system context then it gets the highest preference.
-    String systemPreferExternalIP = systemContext.getProperties().getOrDefault(DataprocConf.PREFER_EXTERNAL_IP,
-                                                                               "false");
-    if (Boolean.parseBoolean(systemPreferExternalIP.trim())) {
-      LOG.debug("System configuration set to prefer external IP. Will use external IPs for DataProc cluster.");
```

Comment:
should we keep this debug statement?

Reply:
This debug message is out of place. If we really want to debug, we should log the whole conf context, but it seems really large to log these (Ideally should be in the audit log of dataproc, need to see if it is available).